db/migrate/20161102222517_add_consent_to_drawings.rb

```diff
@@ -0,0 +1,5 @@
+class AddConsentToDrawings < ActiveRecord::Migration
+  def change
+    add_column :drawings, :image_consent, :boolean
```

Comment:
We should avoid the 3-state boolean problem and simplify things down the line (i.e. we can be sure this value is either true of false). So setting this field to now allow nulls and set the default value of the column to `false` as all existing records would not have gained explicit consent.

This will also remove the ambiguous ? issue above.